libs/core/src/lib/select/select.component.scss

```diff
@@ -80,6 +80,22 @@
                 }
             }
         }
+		&.fd-button--noborder{
+			  border-color: var(--sapButton_Lite_Background, transparent);
```

Comment:
`No border` will always have transparent color. Not even `--sapButton_Lite_Background, ` why would you apply background to this ?